docs/spec/overview.md

```diff
@@ -15,6 +16,15 @@ The interfaces are ([Addressable](interfaces.md#addressable),
 [Subscribable](interfaces.md#Subscribable), [Callable](interfaces.md#callable)).
 For more details, see [Interface Contracts](interfaces.md).
 
+- A **Broker** provides the eventing mesh.
+
+- A **Trigger** describes the filtering and subscription of events from a
```

Comment:
I think using the term "subscription" here is likely to be confusing given "Subscription" type below.

What about "describes a filter on event attributes which should be delivered to an Addressable"?